### ISSUE TITLE
For policy/share etc local FS should display intuitive error

### ISSUE THREAD

**Issue by krishnasrinivas** [minio/mc#1829]
User reported:

```
I'm getting errors while I'm trying to generate share url, but I always get mc: <ERROR> Unable to set policy ‘download’ for ‘storage/default/MyFile1’. ‘SetAccess’ feature is not implemented for ‘filesystem’.Tried on exfat4 and ntfs on ubuntu and windows. Is this behavior okay?
"storage" is just a directory. I run server with minio server storage and those commands don't work:
mc policy download storage/default/MyFile1
mc share download storage/default/MyFile1

@Rasie1 policy/share works on s3 storage, so you have to do:
mc policy download s3alias/bucketName

@krishnasrinivas thank you, for some reason I thought I can use mc on local files that are inside of some bucket

```

user was trying policy/share on local FS, error should have been "the operation is supported on S3, not local FS"

**Comment by vadmeste** [comment]
@krishnasrinivas, What about this one:
$ mc share download /tmp/file
mc: <ERROR> Unable to share a non S3 url ‘/tmp/file’. ‘ShareDownload’ feature is not implemented for ‘filesystem’.

**Comment by krishnasrinivas** [comment]
@vadmeste `feature is not implemented for ‘filesystem’`  indicates that it will be implemented in future, we can say `the operation is not supported for filesystem`

**Comment by harshavardhana** [comment]
> @vadmeste feature is not implemented for ‘filesystem’ indicates that it will be implemented in future, we can say the operation is not supported for filesystem

"Is not supported" is clearly better @krishnasrinivas :+1:

**Comment by harshavardhana** [comment]
Closed by #1833
